pkg/sanity/controller.go

```diff
@@ -1640,7 +1489,6 @@ func MakeCreateVolumeReq(sc *TestContext, name string) *csi.CreateVolumeRequest
 		},
 		CapacityRange: &csi.CapacityRange{
 			RequiredBytes: size1,
-			LimitBytes:    size1,
```

Comment:
Will there be a test with `LimitBytes` set? This change will remove it from all current `MakeCreateVolumeReq` callers and there are quite a lot of them already.

Reply:
I think this is not required by most CSI driver. Kubernetes does not use `LimitBytes`.

I can change `should fail when requesting to create a volume with already existing name and different capacity` test back to set LimitBytes. It needs to ensure the first volume is smaller than `size2`.

Reply:
Maybe we can add a dedicated test that set very small `LimitBytes` (say 1 byte), and assert the CreateVolume would fail.

Reply:
Yes, a single test with `LimitBytes` would ensure we don't lose the test coverage. All the others can use `MakeCreateVolumeReq` as you wrote.